src/shared/bus-polkit.c

```diff
@@ -252,7 +252,7 @@ static int async_polkit_query_new(
                 const char **details,
                 AsyncPolkitQuery **ret)
 {
-        AsyncPolkitQuery *q;
+        _cleanup_(async_polkit_query_freep) AsyncPolkitQuery *q = NULL;
```

Comment:
Ah, you've already done. Please merge the last two commits.

Reply:
Well, the idea was that one commit just moves a block of code around (into a new function) and another commit does changes to it. But if you wish, I'll merge them.

Reply:
Yeah, please.

Reply:
Done.